client/src/modules/cohort/index.jsx

```diff
@@ -128,6 +134,13 @@ class Cohort extends PureComponent {
       : removeArchivedListsMetaData();
   };
 
+  handleListType = isPrivate => {
```

Comment:
This one form field is handle (manage) in different way. Could we pass it directly to handleListCreation link in name/description case?

Reply:
not really, cause then we gonna create a new list in this function and then also setting list type in the same function.
Referring to a single responsibility principle, a function should do only one thing. That's why it's separated.